Wikipedia/Code/WMFLanguageCell.m

```diff
@@ -70,14 +73,28 @@ - (void)awakeFromNib {
     [self prepareForReuse];
     [self wmf_makeCellDividerBeEdgeToEdge];
     self.showsReorderControl = YES;
+    self.primaryLabel.textColor = [UIColor whiteColor];
+    self.isPrimary = NO;
 }
 
 - (void)prepareForReuse {
     [super prepareForReuse];
     self.languageNameLabel.text           = @"";
```

Comment:
Is there a reason we are not setting these to nil?
Usually this is the way we clear a label as opposed to empty strings unless there is a bug you need to work around.